docs/learn/learn-account-generation.md

```diff
@@ -39,98 +40,25 @@ security trade-offs. Please review this page carefully before making your accoun
 understand the risks of the account generation method you choose and how to properly mitigate them
 in order to keep your funds safe.
 
-### Storing your key safely
-
-The seed is your **key** to the account. Knowing the seed allows you, or anyone else who knows the
-seed, to re-generate and control this account.
-
-It is imperative to store the seed somewhere safe, secret, and secure. If you lose access to your
-account (i.e. you forget the password for your account's JSON file), you can re-create it by
-entering the seed. This also means that somebody else can have control over your account if they
-have access to your seed.
-
-For maximum security, the seed should be written down on paper or another non-digital device and
-stored in a safe place. You may also want to protect your seed from physical damage, as well (e.g.
-by storing in a sealed plastic bag to prevent water damage, storing it in a fireproof safe, etching
-it in metal, etc.) It is recommended that you store multiple copies of the seed in geographically
-separate locations (e.g., one in your home safe and one in a safety deposit box at your bank).
-
-:::caution You should not store your seed on any kind of computer that has or may have access to the internet in the future.
+:::info Visit [this support article](https://support.polkadot.network/support/solutions/articles/65000181874-how-to-store-your-mnemonic-phrase-and-backup-file-safely) for more information about key security.
 
 :::
 
-### Storing your account's JSON file
-
-The JSON file is encrypted with a password, which means you can import it into any wallet which
-supports JSON imports, but to then use it, you need the password. You don't have to be as careful
-with your JSON file's storage as you would with your seed (i.e. it can be on a USB drive near you),
-but remember that in this case, your account is only as secure as the password you used to encrypt
-it. Do not use easy to guess or hard to remember passwords. It is good practice to use a
-[mnemonic password of four to five words](https://xkcd.com/936/). These are nearly impossible for
-computers to guess due to the number of combinations possible, but much easier for humans to
-remember.
+## Polkadot JS Browser Extension
 
-## Polkadot{.js} Browser Extension
-
-The Polkadot{.js} Extension provides a reasonable balance of security and usability. It provides a
+The Polkadot JS Extension provides a reasonable balance of security and usability. It provides a
 separate local mechanism to generate your address and interact with Polkadot.
```

Comment:
Staying consistent with Glossary https://support.polkadot.network/support/solutions/articles/65000110873-glossary and at the same time, preserving the link to this section referenced on third party websites
```suggestion
The Polkadot-JS Browser Extension (simply referred to as Polkadot Extension) provides a reasonable balance of security and usability. It provides a
separate local mechanism to generate your address and interact with Polkadot.
```